fix(curriculum): correct explanation about arrow functions and update hint text (#58490)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/660ae3eeef9ad289bece426b.md

() => {}
```

By assigning the arrow function definition to a variable, you bind it to an identifier.

```js
const exampleFunction = () => {
  // code goes here
}
```

Create a new arrow function and assign it to the variable `printGreeting`. Inside the function body, use the `console.log()` method to print the string `Hello there!`.

# --hints--

> You should have a variable named `printGreeting` that stores an arrow function.

Reviewer: ```suggestion
You should have a variable named `printGreeting` and assign it an arrow function.
```

```js
assert.isFunction(printGreeting);
```

> You should use arrow function syntax and store it in the variable `printGreeting`.

Reviewer: ```suggestion
You should create an arrow function and assign it to the `printGreeting` variable.
```

```js
assert.match(code, /(let|const)\s+printGreeting\s*=\s*\(.*\)\s*=>\s*{\s*[\s\S]*}\s*;?/)
```

> The arrow function stored in `printGreeting` should include a `console.log('Hello there!')`.

Reviewer: ```suggestion
Your `printGreeting` function should log "Hello there!" to the console`.
```
